package.json

```diff
@@ -54,7 +54,7 @@
     "markdown-it-jsx": "^1.1.0",
     "markdown-toc": "^1.2.0",
     "minimist": "^1.2.8",
-    "mkdirp": "^1.0.4",
+    "mkdirp": "^3.0.0",
```

Comment:
I have a question, do we need this module?
I can't find the directory where this module is used, so could you test if it works well without this module?

Reply:
The docs are generated correctly without this depencency, so I deleted it